rice-framework/krad-it/src/test/java/org/kuali/rice/krad/service/DateTimeServiceTest.java

```diff
@@ -340,4 +340,28 @@ private static void waitForMidnightIfWithinOneSecond() throws InterruptedExcepti
 
         assertEquals("-5", Integer.toString(CoreApiServiceLocator.getDateTimeService().dateDiff(date1, date2, false)));
     }
+
+    @Test public void testDateDiffDaylightSavingsTimeInclusive() throws ParseException {
```

Comment:
I added some javadoc to the `testDateDiffDaylightSavingsTimeInclusive` test that should clarify things a bit.